apps/wallet-mobile/src/features/Receive/useCases/MultipleReceives.tsx

```diff
@@ -0,0 +1,155 @@
+import {useTheme} from '@yoroi/theme'
+import React, {useState} from 'react'
+import {SafeAreaView, StyleSheet, Text, useWindowDimensions, View} from 'react-native'
+import Animated, {Layout} from 'react-native-reanimated'
+
+import {ModalScreenWrapper} from '../../../../src/components/ModalScreenWrapper/ModalScreenWrapper'
+import {Button, Spacer, StatusBar} from '../../../components'
+import {useHideBottomTabBar} from '../../../yoroi-wallets/hooks'
+import {InfoCard} from '../common/InfoCard/InfoCard'
+import {mocks} from '../common/mocks'
+import {SmallAddressCard} from '../common/SmallAddressCard/SmallAddressCard'
+import {useNavigateTo} from '../common/useNavigateTo'
+import {useStrings} from '../common/useStrings'
+import QRs from '../illustrations/QRs'
+
+interface Item {
+  isUsed: boolean
+  loading: boolean
+}
+
+export function MultipleReceives() {
+  useHideBottomTabBar()
+  const strings = useStrings()
+  const {styles, colors} = useStyles()
+
+  const navigate = useNavigateTo()
+
+  const HEIGHT_SCREEN = useWindowDimensions().height
+  const HEIGHT_MODAL = (HEIGHT_SCREEN / 100) * 70
+
+  const [isModalVisible, setIsModalVisible] = useState(true)
+
+  const [data, setData] = useState(mocks.addressList)
+
+  const onRequestAddress = () => {
```

Comment:
handleOnRequest... or if single command use the `navigate.receiveDetails` direct on `onPress...`

Reply:
◀️